src/electron/platform/android/setup/live-android-service-setup-business-logic.ts

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { AndroidServiceApkLocator } from 'electron/platform/android/android-service-apk-locator';
+import {
+    AndroidServiceConfigurator,
+    DeviceInfo,
+    PackageInfo,
+} from 'electron/platform/android/android-service-configurator';
+
+export interface AndroidServiceSetupBusinessLogic {
```

Comment:
What problem is this interface solving? There's only one implementation, and it's exactly the same code to mock it for a test with or without the extra interface in between, no? Why the extra layer?

Reply:
Muscle memory from C# mocking libraries. Will remove the extra layer